src/python/WMCore/Credential/Proxy.py

```diff
@@ -174,13 +174,17 @@ def setEnv(self, cmd):
     def cmd_exists(self, cmd):
         return subprocess.call(self.setEnv("type " + cmd), shell=True,
                                stdout=subprocess.PIPE, stderr=subprocess.PIPE) == 0
+    def getUserCertFilename(self):
+        ucFilename = ('~/.globus/usercert.pem' if 'X509_USER_CERT'
```

Comment:
Sorry Kenyi, I meant:
```
if 'X509_USER_CERT' not in os.environ:
    ucFilename = '~/.globus/usercert.pem'
else:
    ucFilename = os.environ['X509_USER_CERT']
return ucFilename
```

It's a very small method, so the "classic" if-else style looks better IMO.

Reply:
Done